modules/smtp/modules.php

```diff
@@ -1540,6 +1557,105 @@ protected function output() {
     }
 }
 
+/**
+ * Send scheduled messages
+ * @subpackage smtp/handler
+ */
+class Hm_Handler_send_scheduled_messages extends Hm_Handler_Module {
+    /**
+     * Send delayed messages
+     * This should use cron
+     */
+   public function process() {
+    if (!($this->module_is_supported('imap') || $this->module_is_supported('profiles'))) {
+        return;
+    }
+
+    $servers = Hm_IMAP_List::dump();
+    $scheduled_msg_count = 0;
+
+    foreach (array_keys($servers) as $server_id) {
+        $cache = Hm_IMAP_List::get_cache($this->cache, $server_id);
+        $imap = Hm_IMAP_List::connect($server_id, $cache);
```

Comment:
This is no longer the way to get a connection to imap server. Imap, Jmap and EWS were unified to work behind a bridge called Mailbox. See other handler modules for examples how to retrieve a mailbox and work with it. Basically, anything that works with IMAP should work with EWS as well if you go through the bridge.